<ui version="4.0" >
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Find Text</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Text:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textEdit_" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="backwardsCheck_" >
        <property name="text" >
         <string>Find Backwards</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="caseSensitiveCheck_" >
        <property name="text" >
         <string>Case Sensitive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="wholeWordsCheck_" >
        <property name="text" >
         <string>Whole Words Only</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton_" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findNextButton_" >
       <property name="text" >
        <string>Find &amp;Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton_" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton_</sender>
   <signal>clicked()</signal>
   <receiver>FindDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>203</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel" >
     <x>102</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton_</sender>
   <signal>clicked()</signal>
   <receiver>FindDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>344</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel" >
     <x>304</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
